src/android/build.gradle

```diff
@@ -8,7 +8,7 @@ dependencies {
      *
```

Comment:
This must help:
```
repositories {
         maven { url 'https://jitpack.io' }
}
```

Reply:
@Proper-Job, does it help?

Reply:
It works with capacitor this way but not with cordova. Debugging right now.

Reply:
```groovy
allprojects {
    repositories {
        maven { url 'https://jitpack.io' }
    }
}
```
seems to do the trick for both capacitor and cordova, though.

Reply:
@exxbrain I pushed your change request. I will test this again with a clean cordova and capacitor install.

Reply:
@exxbrain @NiklasMerz I successfully tested cordova and capacitor integration with these latest changes on a real Android device. No additional configuration was necessary after installing the plugin.